lib/app.js

```diff
@@ -51,6 +51,61 @@ app.get('/videogames/:id', async(req, res) => {
   }
 });
 
+// name: "Demon's-Souls",
+// system: "PlayStation3",
+// played: true,
+// year_released: 2011
+
+app.post('/videogames', async(req, res) => {
+  const { name, system, played, year_released } = req.body;
+
+  try {
+    const videogamesData = await client.query(`
+    INSERT INTO videogames(
+      name,
+      system,
+      played,
+      year_released
+    ) VALUES ($1, $2, $3, $4)
+    RETURNING *`, [
+      name,
+      system,
+      played,
+      year_released
+    ]);
+    
+    res.json(videogamesData.rows[0]);
+  }catch(e){
+    res.status(500).json({ error: e.message });
+  }
+});
+
+app.put('/videogames/:id', async(req, res) =>{
+  const { id, name, system, played, year_released } = req.body;
+  try{
+    const videogamesData = await client.query(`
+    UPDATE videogames
+    SET
```

Comment:
Fun fact about SET - you actually only need to include the column that you will be changing the data in. So if you only want to change the played value for this game, you can just submit the `played` variable and all other columns will remain the same. 

Just NEVER forget the `WHERE` clause because that is what tells the DB which entries to update.